src/features/wallet/helpers/checkAvailableRewards.ts

```diff
@@ -0,0 +1,30 @@
+import { ethers } from "ethers";
+import { VoteRevealed } from "web3/get/queryVotesRevealedEvents";
+import { queryRetrieveRewards } from "web3/get/queryRetrieveRewards";
+
+export default async function checkAvailableRewards(
+  data: VoteRevealed[],
+  address: string,
+  contract: ethers.Contract
+) {
+  const promises = data.map(async (vote) => {
+    const rewardAvailable = await queryRetrieveRewards(
+      contract,
+      address,
+      vote.roundId,
+      vote.identifier,
+      vote.time
+    );
+    if (rewardAvailable) return parseFloat(rewardAvailable);
```

Comment:
hopefully this doesnt affect reward getting, and only display, but this can cause some information loss / dust if value is used for claiming (which it doesnt look like it does).

Reply:
Yeah this is just for the balance in the 3rd column. It should line up for actual values available, but it could be mismatched. But it won't matter -- the actual blockchain call will determine that.